src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,7 +6,7 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address extends Contact {
```

Comment:
Not sure how this uses its parent class though. `value` in `Contact` is not set because both classes have their own `value` attribute. They are also no getters for any of the attributes in `Contact`

Reply:
My mistake. Have fixed this in new commit https://github.com/nus-cs2103-AY1718S2/addressbook-level2/pull/772/commits/9cc15e38b3adbd500f025150d33a7ddf9beb7385